package.json

```diff
@@ -15,12 +21,27 @@
     "overlap",
     "area",
     "difference",
-    "geometry"
+    "geometry",
+    "intersection"
   ],
   "author": "Ophir LOJKINE",
-  "license": "GPLv2",
+  "license": "LGPL-3.0-or-later",
   "bugs": {
     "url": "https://github.com/lovasoa/rectangle-overlap/issues"
   },
-  "homepage": "https://github.com/lovasoa/rectangle-overlap"
+  "homepage": "https://github.com/lovasoa/rectangle-overlap",
+  "devDependencies": {
+    "@stryker-mutator/core": "^2.0.0",
```

Comment:
I did'nt know stryker. It looks like a very useful testing library.

Reply:
It is, however, be prepared to wait for a long time if you have lots of code to test.